code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -148,11 +151,11 @@
 	var/setting = 0
 
 /obj/item/pinpointer/advpinpointer/process()
-	if(setting == 0)
+	if(setting == SETTING_DISK)
```

Comment:
Can you also make this a switch (again, apologies for mass nitpicks)

Reply:
Tis fine, one moment

Reply:
Switched things up